.circleci/config.yml

```diff
@@ -189,6 +189,44 @@ jobs:
       - store_artifacts:
           path: /tmp
 
+  e2e-nginx:
```

Comment:
Please don't add more suites - we're out of capacity. If it's a large test: prow.
If it's fast - add it to simple or pilot suites.